ome/resource_discovery.go

```diff
@@ -48,6 +53,74 @@ func (r *discoveryResource) Schema(_ context.Context, _ resource.SchemaRequest,
 	}
 }
 
+func (r *discoveryResource) ValidateConfig(ctx context.Context, req resource.ValidateConfigRequest, resp *resource.ValidateConfigResponse) {
+	var data models.OmeDiscoveryJob
+	resp.Diagnostics.Append(req.Config.Get(ctx, &data)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	if data.Schedule.ValueString() == "RunNow" {
+		if !data.Cron.IsNull() {
+			resp.Diagnostics.AddAttributeError(
+				path.Root("cron"),
+				"Attribute Error",
+				"With Schedule as RunNow, CRON can't be set.",
+			)
+		}
+	}
+
+	if len(data.DiscoveryConfigTargets) > 0 {
+		for idx, dct := range data.DiscoveryConfigTargets {
+			idxError := "Inappropriate value for attribute `discovery_config_targets`: element: " + strconv.Itoa(idx) + "\n"
```

Comment:
Quotes are preferred over backticks in errors.

Reply:
replacing ` to \"

Reply:
done